overcloud.yml

```diff
@@ -258,6 +258,15 @@
           deployment_timeout: 2400
         when: scale_compute_vms == true
 
+      - name: change the default values for overcloud-node-growvols
```

Comment:
I think it would be good to add a condition check for `when: osp_release|int >=17` here as the growvols playbook is not there in older OSP releases.

Reply:
ack, i'll make the change